src/components/CreateArea.jsx

```diff
@@ -4,46 +4,37 @@ import AddIcon from '@mui/icons-material/Add';
 import Zoom from '@mui/material/Zoom';
 
 function CreateArea(props) {
-  const [inputText, setInputText] = useState({
-    title: '',
-    content: '',
-  });
+  const [titleInput, setTitleInput] = useState('');
+  const [contentInput, setContentInput] = useState('');
+
   const [titleClicked, setTitleClicked] = useState(false);
 
-  function handleChange(e) {
-    const { name, value } = e.target;
-    setInputText((prevNote) => ({
-      ...prevNote,
-      [name]: value,
-    }));
-  }
   function handleButtonClick(e) {
     e.preventDefault();
-    props.onClick(inputText);
-    setInputText({ ...inputText, title: '', content: '' });
-  }
-  function handleTitleClick() {
-    setTitleClicked(true);
+    props.onClick({ title: titleInput, content: contentInput });
```

Comment:
If you decide to implement my proposals for lines 7 and 8 in this piece of code, you may use the cool JS feature object initializer shorthand". https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions
```
 props.onClick({ title, content });
 ```